From cb08367f7f2f72744cfd44407f736ce6e0e0dfee Mon Sep 17 00:00:00 2001
From: Jay Logue <jay.logue@gmail.com>
Date: Mon, 20 Apr 2026 10:30:54 -0700
Subject: [PATCH] package/am335x-pru-package: update version

-- Updated version of package/am335x-pru-package to commit 1e9ff57 (2018/05/18)

Signed-off-by: Jay Logue <jay.logue@gmail.com>
---
 package/am335x-pru-package/am335x-pru-package.hash | 2 +-
 package/am335x-pru-package/am335x-pru-package.mk   | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/package/am335x-pru-package/am335x-pru-package.hash b/package/am335x-pru-package/am335x-pru-package.hash
index 157fa566eb..f85dfddb91 100644
--- a/package/am335x-pru-package/am335x-pru-package.hash
+++ b/package/am335x-pru-package/am335x-pru-package.hash
@@ -1,3 +1,3 @@
 # Locally computed
-sha256  bd90d57d210c2ec1835d701c5d12e228801cb41afebf591e2c146a4dc49c296f  am335x-pru-package-5f374ad57cc195f28bf5e585c3d446aba6ee7096-git4.tar.gz
+sha256  fc6843158866aa2febd8d470c92f50ea3b2198e10a3aa37fb038da872d6606a8  am335x-pru-package-1e9ff57815b84af028451d23e37482ced711c098-git4.tar.gz
 sha256  f0fcdf9b2090896389eb4b784f23be96d5544c5ce5282d84f82ae9a8e8331beb  pru_sw/utils/LICENCE.txt
diff --git a/package/am335x-pru-package/am335x-pru-package.mk b/package/am335x-pru-package/am335x-pru-package.mk
index 9c65ee1387..392d2a1613 100644
--- a/package/am335x-pru-package/am335x-pru-package.mk
+++ b/package/am335x-pru-package/am335x-pru-package.mk
@@ -4,7 +4,7 @@
 #
 ################################################################################
 
-AM335X_PRU_PACKAGE_VERSION = 5f374ad57cc195f28bf5e585c3d446aba6ee7096
+AM335X_PRU_PACKAGE_VERSION = 1e9ff57815b84af028451d23e37482ced711c098
 AM335X_PRU_PACKAGE_SITE = https://github.com/beagleboard/am335x_pru_package
 AM335X_PRU_PACKAGE_SITE_METHOD = git
 AM335X_PRU_PACKAGE_LICENSE = BSD-3-Clause
-- 
2.43.0

